package.json

```diff
@@ -368,44 +337,41 @@
     "@opam/charInfo_width": "*",
     "@opam/dune": "2.5.0",
     "@opam/dune-configurator": "*",
+    "@opam/fmt": "^0.8.8",
+    "@opam/fp": "*",
+    "@opam/fs": "*",
+    "@opam/logs": "^0.7.0",
+    "@opam/luv": "~0.5.1",
     "@opam/lwt": "*",
+    "@opam/markup": "^0.8.2",
     "@opam/ocamlbuild": "*",
     "@opam/ppx_deriving": "*",
     "@opam/ppx_deriving_yojson": "*",
+    "@opam/ppx_inline_test": "v0.13.1",
     "@opam/ppx_let": "v0.13.0",
```

Comment:
Sorry about the mess. I used `esy add` for this, which went to town on the formatting. I actually had no idea that sub-command existed, but felt it should so I tried, and voila. Much more convenient than adding dependencies manually, so I guess this is the formatting we ought to try to maintain.

Reply:
Neat, I forgot about that command too. Maintain this formatting sounds good.